humilis_elasticache/outputs.yaml.j2

```diff
@@ -11,19 +11,23 @@ outputs:
         Value:
             "Fn::GetAtt":
                 - CacheCluster
-                {% if engine=='memcached' %}
-                - ConfigurationEndpoint.Address
-                {% elif engine=='redis' %}
+                {% if engine=='memcached' or redis_cluster_mode %}
```

Comment:
Each of the three case outputs the address and port under a slightly different name.